src/formatDate.js

```diff
@@ -9,6 +9,38 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
-}
+  const data = date.split(fromFormat[3]);
+
+  const object = {};
+
+  for (let i = 0; i < 3; i++) {
+    object[fromFormat[i]] = data[i];
+  }
+
+  if (object.YYYY) {
+    object.YY = object.YYYY.slice(2);
+  }
+
+  if (object.YY < 30) {
+    object.YYYY = '20' + `${object.YY}`;
+  } else {
+    object.YYYY = '19' + `${object.YY}`;
+  }
 
+  const objectKeys = Object.keys(object);
+  const objectValues = Object.values(object);
```

Comment:
There is no need to create separate arrays for keys and values of the `object`. You can directly access the values using the keys when constructing the `result` string.